engine/execution/computation/computer/computer.go

```diff
@@ -201,8 +201,7 @@ func (e *blockComputer) executeSystemCollection(
 
 	serviceAddress := e.vmCtx.Chain.ServiceAddress()
 	tx := fvm.SystemChunkTransaction(serviceAddress)
-	txMetrics := fvm.NewMetricsCollector()
-	err := e.executeTransaction(tx, colSpan, txMetrics, collectionView, programs, e.systemChunkCtx, txIndex, res)
+	err := e.executeTransaction(tx, colSpan, collectionView, programs, e.systemChunkCtx, txIndex, res)
```

Comment:
I defined a new interface type, so we don't have to deal with internal structs of fvm.